src/figma/brandColors.json

```
@@ -608,5 +554,21 @@
       "parent": "Brand Colors/v1 - current",
       "description": ""
     }
+  },
+  "white": {
+    "white": {
+      "value": "#ffffff",
+      "type": "color",
+      "parent": "Brand Colors/v1 - current",
+      "description": ""
+    }
+  },
+  "black": {
+    "black": {
+      "value": "#000000",
+      "type": "color",
+      "parent": "Brand Colors/v1 - current",
+      "description": ""
+    }
```

I've updated this section so that the Figma colors will render in storybook but it's likely we need to update the `useJsonColor` hook cc @garrettbear would you be able to add the updates you made to handle these colors?

```suggestion
  "white": {
    "value": "#ffffff",
    "type": "color",
    "parent": "Brand Colors/v1 - current",
    "description": ""
  },
  "black": {
    "value": "#000000",
    "type": "color",
    "parent": "Brand Colors/v1 - current",
    "description": ""
  }
```